### PR TITLE
Improve jail management

### PR DESCRIPTION
**Pull request by Josue-T**

### CODE REVIEW

File: conf/monitorix.conf [hunk 1]
```diff
@@ -586,8 +586,12 @@ secure_log_date_format = %b %e
 <fail2ban>
         list = Security, Overload / Abuse
```
Comment (supermamie) [inline conf/monitorix.conf:587]: The `list` configuration should have the same number of items as the `desc` list
```suggestion
        list = Overload / Abuse, YunoHost, Mail, Default, SSH, Apps
```

File: scripts/_common.sh [hunk 2]
```diff
@@ -33,6 +33,18 @@ config_nginx() {
 }
 
 config_monitorix() {
+    jail_list=$(fail2ban-client status | grep 'Jail list:' | sed 's/.*Jail list://' | sed 's/,//g')
+    additional_jail=""
+    for jail in $jail_list; do
+        if ! [[ "$jail" =~ (recidive|pam-generic|yunohost|postfix|postfix-sasl|dovecot|nginx-http-auth|sshd|sshd-ddos) ]]; then
```
Comment (supermamie) [inline scripts/_common.sh:39]: Can this test be done also for the "default" rules ?
Just to be sure that they exist

Comment (Josue-T) [reply scripts/_common.sh:39]: Don't really understand what you mean

Comment (supermamie) [reply scripts/_common.sh:39]: Can we be sure that everyone have jails named recidive|pam-generic|yunohost|postfix|postfix-sasl|dovecot|nginx-http-auth|sshd|sshd-ddos ?
If one is not active, there will be massive logs about that.

Comment (Josue-T) [reply scripts/_common.sh:39]: Normally they should exist as it should be provided by default on Yunohost. I'll make a deeper check.

**Pull request merged**